src/ui/components/shared/SharingModal/CollaboratorsList.css

```
@@ -6,6 +6,19 @@
   max-height: 240px;
 }
 
+.sharing-modal .permissions-list {
```

I don't think we need this nesting/specificity, do we?

Nice to avoid it as much as possible.
```suggestion
.permissions-list {
```